hz.ps1

```diff
@@ -555,6 +555,9 @@ function ensure-jar ( $jar, $repo, $artifact ) {
     $classpath = $script:options.classpath
     if (-not [System.String]::IsNullOrWhiteSpace($classpath)) { $classpath += $s }
     $classpath += "$tmpDir/lib/$jar"
+    # condiser to quote the path to escape from white space 
```

Comment:
You probably want this to be "consider" ;) And, maybe it's even "be sure to..."?

Reply:
You are right, that could better :)